apps/src/gamelab/GameLab.js

```diff
@@ -1063,6 +1072,46 @@ GameLab.prototype.onP5Draw = function () {
   this.completeRedrawIfDrawComplete();
 };
 
+/**
+ * Capture a thumbnail image of the play space if the app has been running
+ * for long enough and we have not done so already.
+ */
+GameLab.prototype.captureInitialImage = function () {
+  if (!project.isOwner() || this.initialCaptureComplete || this.tickCount < CAPTURE_TICK_COUNT) {
+    return;
+  }
+  this.captureImage(THUMBNAIL_SIZE);
+  this.initialCaptureComplete = true;
```

Comment:
I wonder what we can test here - it would be nice to know right away if this stopped happening for some reason.